js/chat_app/src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable jsx-quotes */
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { setStatus } from './redux/slicies/testSlise';
 
 function App() {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.test);
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <p>{status.toString()}</p>
+      <button type='submit' onClick={() => dispatch(setStatus())}>
```

Comment:
Since we're using MaterialUI, i would ask you to use Buttons from package already